acceptance_tests/test_register_page.py

```diff
@@ -1,49 +1,46 @@
-import unittest
+from django.test import TestCase
 
 from page_objects import PageObject, PageElement
 from selenium import webdriver
 
 
-class RegisterPage(PageObject):
-    username = PageElement(name="username")
-    password1 = PageElement(name="password1")
-    password2 = PageElement(name="password2")
-    register = PageElement(class_name="submit-register")
+class RegistrationPage(PageObject):
+    username = PageElement(name='username')
+    password1 = PageElement(name='password1')
+    password2 = PageElement(name='password2')
```

Comment:
I think you can apply the name suggestions from the 1st round of reviews since these attributes are only used in tests and are not in any way bound with the form.